homeassistant/components/unifi/__init__.py

```diff
@@ -28,7 +28,7 @@ async def async_setup(hass, config):
 
 
 async def async_setup_entry(hass, config_entry):
-    """Set up the UniFi component."""
+    """Set up the UniFi Network component."""
```

Comment:
So this is a bit of a HA thing. But do we want reference the module here as a "component" (how HA represents integrations inside of the code) or as an "integration" (how HA represents them inside of the frontend).

I did not change that since I figured comments in the code should refer to it as a component (but I do not care either way, which ever you prefer).

Reply:
I was just into correcting stuff :)

Reply:
That is fine. That is just why I was asking for clarification. Do you want to reference everything in code as an "integration"? "component" and "integration" are a bit interchangeable in the code.

Reply:
Yes, component is really the old naming scheme

Reply:
Went through all of the component instances I found and renamed them integration. Resolving these ones so I can double check and compare.

I will push it all up once you response to the other comment about the controller vs. application bit.